ksa_compliance/standard_doctypes/tax_category.py

```diff
@@ -0,0 +1,101 @@
+import frappe
+
+
+def tax_category_mapper(tax_category_id: str) -> dict:
```

Comment:
Create a dataclass to hold the result instead of relying on dictionaries for public functionality (i.e. any function that is meant to be used outside its defining module). This helps loose coupling and ensures type analysis can catch usage errors.